WordPress/Classes/ViewRelated/Media/SiteMedia/Views/SiteMediaCollectionCellSelectionOverlayView.swift

```diff
@@ -28,14 +27,7 @@ final class SiteMediaCollectionCellSelectionOverlayView: UIView {
     func setBadge(_ badge: SiteMediaCollectionCellViewModel.BadgeType) {
         switch badge {
         case .unordered:
-            badgeView.textLabel.attributedText = NSAttributedString(attachment: {
-                let attachment = NSTextAttachment()
-                let configuration = UIImage.SymbolConfiguration(font: UIFont.systemFont(ofSize: 11, weight: .semibold))
-                attachment.image = UIImage(systemName: "checkmark", withConfiguration: configuration)?.withTintColor(.white, renderingMode: .alwaysTemplate)
-                return attachment
-            }(), attributes: [
-                NSAttributedString.Key.baselineOffset: 1 // It doesn't appear visually centered othwerwise
-            ])
+            badgeView.textLabel.text = "✓"
```

Comment:
The icon was used as the white check inside the green circle background

| Before | After |
| ------ | ----- |
| <img width="556" alt="check-before" src="https://github.com/user-attachments/assets/293fac19-9b84-47a6-a8c4-f637e34bfad0"> | <img width="556" alt="check-after" src="https://github.com/user-attachments/assets/a6901489-aaf1-42b8-884c-e2f3b5d1fd79"> |